pom.xml

```diff
@@ -15,12 +15,12 @@
     <doclint>none</doclint>
     <driver.version>4.9.0</driver.version>
     <xml-format.skip>true</xml-format.skip>
-    <dropwizard.version>2.0.21</dropwizard.version>
+    <dropwizard.version>2.0.26</dropwizard.version>
```

Comment:
I think I remember when we were doing this before, other dependencies should also be checked and I think updated.. Have the feeling that this is not enough..

Reply:
Furthermore I see you are merging V2 PR without approvals, what's the reason for it? 

I do understand you want to be faster, but that's eventually gonna be merged to master and I don't think we will do a big review then, but rather count that all in V2 branch was reviewed.. Or what's the idea?

Reply:
The idea is probably not to merge to `master`/`mainline` any more (although that was considered at some point), but that it will replace master/mainline as for v2.0.0.

I try to wait for approval in general, but in this case I wanted to unblock Jeff -- although in hindsight I could just have asked him to approve it.

Reply:
On DropWizard version: I updated dw-metrics as suggested. It is possible that jetty version could use upgrade; I'll check that now. If you remember others that likely need upgrade, please let me know.

Reply:
> The idea is probably not to merge to master/mainline any more

If the `v2.0.0` branch isn't going to be merged then how will all of this be reviewed?

Reply:
Right, there needs to be a review process closer to when we consider work not to be prototype any more.
Regardless of whether there is actual merge (which may or may not be feasible) or simply switching mainline to be another branch.

The reason I think merge/non-merge is not a big issue is because so far very little code would change in merge-style way: almost everything is technically new code (under new modules) and there are few changes to SGv1 codebase itself. And of SGv1 changes made some have been backported.

Perhaps this isolation will change after REST module (except for Documents API that likely will be extracted to another module); if so we will need to decide if we do actually want a merge. My assumption was simply that merging has limited value & not merging leaves as fully functional V1 for however long anyone wants it (granted this can be handled by branching separate `v1` from `master` before merge so maybe that's a moot point).

Reply:
* +1 for merging and creating a `v1` prior to that
* +1 for also having protection rules on the `v2` branch asap